extension/content/components/pages/RecipesPage.js

```diff
@@ -197,6 +198,7 @@ class RecipesPage extends React.PureComponent {
       this.setState({ runningArbitrary: false });
     }
   }
+  /* eslint-enable no-useless-catch */
```

Comment:
this should actually go up above on line 200 above the last `}`.

Reply:
will do